drivers/flash/flash_shell.c

```diff
@@ -156,7 +156,11 @@ static int cmd_read(const struct shell *shell, size_t argc, char *argv[])
 	while (cnt--) {
 		uint32_t data;
 
-		flash_read(flash_dev, addr, &data, sizeof(data));
+		ret = flash_read(flash_dev, addr, &data, sizeof(data));
+		if (ret != 0) {
+			shell_error(shell, "Read ERROR!");
```

Comment:
It'd be nice to have this be:

    shell_error(shell, "Read failed, code %d", result);

which is a more informative style of diagnostic used in a few other places in this file.

Reply:
@nzmichaelh ^^